crowbar_framework/app/models/batch/export.rb

```diff
@@ -17,6 +17,8 @@
 
 module Batch
   class Export < Base
+    attr_accessor :path
```

Comment:
If you want to use a custom file name then yes, otherwise it makes not really sense if we don't use it.